api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/redis/BaseRedisProperties.java

```diff
@@ -64,6 +64,19 @@
      */
     private boolean useSsl;
 
+    /**
+     * Setting that describes how Lettuce routes read operations to replica nodes.
+     * The default mode is MASTER, it will read from the current master node.
+     * Other mode are :
+     * MASTER_PREFERRED: Read from the master, but if it is unavailable, read from replica nodes.
+     * REPLICA/SLAVE: Read from replica nodes. The value REPLICA should be used since lettuce-core version 5.2.
+     * REPLICA_PREFERRED/SLAVE_PREFERRED: Read from the replica nodes, but if none is unavailable, read from the
+     * master. The value REPLICA_PREFERRED should be used since lettuce-core version 5.2
+     * NEAREST: Read from any node of the cluster with the lowest latency.
+     * ANY: Read from any node of the cluster.
```

Comment:
Very nice. Bonus points if you could also wrap values in `<code>` tags and perhaps each item inside a `<li>` element. Example: https://github.com/apereo/cas/blob/master/api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/ldap/AbstractLdapAuthenticationProperties.java#L22

Reply:
Ok I will try to obtain some bonus points :smile: